src/helpers.js

```diff
@@ -2,16 +2,40 @@ import MutationObserver from '@sheerun/mutationobserver-shim'
 
 const globalObj = typeof window === 'undefined' ? global : window
 
+function runWithRealTimers(callback) {
+  const usingJestFakeTimers = globalObj.setTimeout._isMockFunction && !!jest
```

Comment:
I think this would need to be:

```javascript
globalObj.setTimeout._isMockFunction && typeof jest !== 'undefined'
```

Otherwise I _think_ we'd get an `Uncaught ReferenceError: jest is not defined` in other frameworks.

Reply:
Good catch!